changelog.d/pr-7346.md

```diff
@@ -0,0 +1,4 @@
+### 🏠 Internal
+
+- Remove `__init__()` from `GeneratorMixIn` and `AssemblingDecoderMixIn` to convert them to "proper" mixin-classes.  [PR #7346](https://github.com/datalad/datalad/pull/7346) (by [@christian-monch](https://github.com/christian-monch))
+- Fix spelling errors and add `includeds` to codespell's ignore words list .  [PR #7346](https://github.com/datalad/datalad/pull/7346) (by [@christian-monch](https://github.com/christian-monch))
```

Comment:
```suggestion
```
since not pertinent to this PR and was added already in main codebase in 6c421030929498495e0229edb55efa0abad1139e awhile back